app/assets/javascripts/application.js

```diff
@@ -14,3 +14,43 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+
+// Clicking 'Submit' button for a New Round
+
+$(document).ready(function() {
+  $(document).on('submit', '.new-round-form form', function(event) {
+    event.preventDefault();
+    var form = $(this)
+    $.ajax(form.attr('action'), {
+    type: 'post',
+    data: form.serialize(),
+    success: function(newRound) {
+      $('.rounds').append(newRound);
+      form.find('input').val("");
+      form.find('#round_comments').val("");
+      form.find('.submit-button').val("Submit");
+    }
+    })
+  })
+})
+
+// Clicking 'Delete' button
+
+$(document).ready(function() {
+  $(document).on('click', '.delete-button', function(event) {
+    event.preventDefault();
+    confirm("Are you sure you want to delete this Round?");
+    var deleteButton = $(this)
```

Comment:
Doesn't seem like you're actually doing the AJAX? Not sure why the blank page is happening, but that is worth looking at.